doc/introduction/circuits.rst

```diff
@@ -167,8 +172,58 @@ For example:
     result = circuit(0.543)
 
 
-Quantum circuits from other frameworks
---------------------------------------
+Collections of QNodes
+---------------------
+
+Sometimes you may need multiple QNodes in your variational algorithm. This is
+as simple as defining multiple QNodes as above, using the same or multiple
+devices, and composing them together.
+
+PennyLane also provides some high-level tools for creating and evaluating
+QNode 'collections'. For example, :func:`~.map` allows a single
+function of quantum operations (or :doc:`template <templates>`) to be mapped across
+multiple observables or devices.
+
+For example, consider the following quantum function ansatz:
+
+.. code-block:: python
+
+    def my_ansatz(params, **kwargs):
+        qml.RX(params[0], wires=0)
+        qml.RX(params[1], wires=1)
+        qml.CNOT(wires=[0, 1])
+
+We can define a list of observables, and two devices:
+
+>>> obs_list = [qml.PauliX(0) @ qml.PauliZ(1), qml.PauliZ(0) @ qml.PauliX(1)]
+>>> qpu1 = qml.device("forest.qvm", device="Aspen-4-4Q-D") # requires PennyLane-Forest
+>>> qpu2 = qml.device("forest.qvm", device="Aspen-7-4Q-B") # requires PennyLane-Forest
+
+Note that the two devices above require the `PennyLane-Forest plugin <https://pennylane-forest.rtfd.io>`_
+be installed, as well as the Forest QVM. You can also try replacing them with alternate devices.
+
+Mapping the template across the observables and devices creates a :class:`~.QNodeCollection`:
+
+>>> qnodes = qml.map(my_ansatz, obs_list, [qpu1, qpu2], measure="expval")
+>>> params = [0.54, 0.12]
```

Comment:
Add:
```
>>> qnodes.type
QNodeCollection
```

or so , to show what type "qnodes" is?  

Then say "QNodes in QNodeCollections have the same signature, and we can therefore feed them the same parameters" or so?

Reply:
Have added this in!